calicoctl/calico_ctl/node.py

```diff
@@ -260,8 +261,8 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
     the global default value will be used.
     :param detach: True to run in Docker's "detached" mode, False to run
     attached.
-    :param libnetwork_image: The name of the Calico libnetwork driver image to
-    use.  None, if not using libnetwork.
+    :param libnetwork_enabled: True to run libnetwork plugin inside calico-node.
+    False by default. None, if not using libnetwork.
```

Comment:
So it doesn't start libnetwork for the systems like kubernetes where libnetwork is not used, and the flag is not passed